test/end-to-end/utils/coverage.js

```diff
@@ -7,18 +7,35 @@
 // collects coverage from the tested page,
 // calls nightmare.end() and calls done()
 // executed via eval() as a last step of every test!
+if (pageConfig.root.includes('9090')) {
```

Comment:
I don't like this high-level if statement and then repeating the same code block twice.